src/app/modules/ix-forms/classes/simple-async-combobox-provider.ts

```diff
@@ -31,5 +31,7 @@ export class SimpleAsyncComboboxProvider implements IxComboboxProvider {
   }
 
   private options: Option[];
-  constructor(private options$: Observable<Option[]>) { }
+  constructor(private options$: Observable<Option[]>) {
+    options$.pipe(take(1)).subscribe((value) => this.options = value);
```

Comment:
This is incorrect, this class is not designed to load things on its own. If you are concerned about combobox constantly spinning in some forms, then this is another bug: https://ixsystems.atlassian.net/browse/NAS-120116